delegatedSending.js

```diff
@@ -43,16 +46,101 @@ module.exports.canSend = internal.canSend
 module.exports.getPool = internal.getPool
 module.exports.invitingFromPool = internal.getPool
 
+/**
+ * Get the stars that have put invites into the pool
+ * @param {Number} pool - Pool number
+ * @return {Promise<Array<Number>>} Stars that touched the pool
+ */
+module.exports.getPoolStars = internal.getPoolStars
+
 /**
  * Returns true if receipients is eligible to receive a point, false otherwise
  * @param {String} recipient - Ethereum address
  * @return {Promise<Bool>} Whether recipient can receive a point
  */
 module.exports.canReceive = internal.canReceive
 
+/**
+ * Returns the total amount of invites (across all stars) available to point
+ * @param {Number} point - The point whose invites to count
+ * @return {Promise<Number>} Total amount of invites
+ */
```

Comment:
@jtobin Starting to think this should maybe also do the "ignore improperly configured stars" thing that `getPlanetsToSend` does, so you get a count of total _usable_ invites rather than total _theoretical_ invites, which is often the number clients want to use, especially since `getPlanetsToSend` does bound on that.

Seems sane y/n?

Reply:
Sanity agreed!